### ISSUE TITLE
Improve input filtering

### ISSUE THREAD

**Issue by tonin** [perfsonar/toolkit#238]
There are some places where we should improve user input filtering as that can cause configuration file error parsing.

From GN4-SA2T1 report 2.2.15.

**Comment by apertome** [comment]
This is pretty vague, are more details available somewhere?

**Comment by tonin** [comment]
Yes, sorry for being too vague, I didn't wanted to discolse too much information.  There is a complete example in section 2.2.15 of this doc: https://lists.internet2.edu/sympa/arc/perfsonar-developer/2017-04/msg00000/perfSonar_best_practices_report_final.docx  It is in the test configuration pages.

I'm not sure if this is still applicable as-is as we're now using pscheduler, but probably better to double check.  I will try to reproduce.
